android/brave_java_sources.gni

```diff
@@ -256,4 +256,12 @@ brave_java_base_sources = [
   "//brave/android/java/org/chromium/base/BraveReflectionUtil.java",
 ]
 
+brave_java_base_module_sources = [
+  "//brave/android/java/org/chromium/chrome/browser/base/SplitCompatJobIntentService.java",
+  "//brave/android/java/org/chromium/chrome/browser/upgrade/BraveUpgradeJobIntentService.java",
```

Comment:
is this supposed to be duplicated here and not moved from above?

Reply:
No the file names are different.

Reply:
ah, I see. Thanks